src/high-level/src/KinDynComputations.cpp

```diff
@@ -1073,6 +1073,26 @@ Transform KinDynComputations::getWorldTransform(const FrameIndex frameIndex)
     return world_H_frame;
 }
 
+std::vector<iDynTree::Matrix4x4> KinDynComputations::getWorldTransformsAsHomogeneous(std::vector<std::string> frameNames)
+{
+    std::vector<iDynTree::Matrix4x4> worldTransforms;
+    int numberOfTransforms=frameNames.size();
+    for(int number=0;number<numberOfTransforms;number++)
```

Comment:
```suggestion
    for(int number=0; number<numberOfTransforms; number++)
```
please follow the stlye of the  rest of the code, thanks.